packages/tracing/src/browser/browsertracing.ts

```diff
@@ -266,13 +261,3 @@ export function getMetaContent(metaName: string): string | null {
   const el = getGlobalObject<Window>().document.querySelector(`meta[name=${metaName}]`);
   return el ? el.getAttribute('content') : null;
 }
-
-/** Adjusts transaction value based on max transaction duration */
-function adjustTransactionDuration(maxDuration: number, transaction: IdleTransaction, endTimestamp: number): void {
```

Comment:
we can remove `maxTransactionDuration` because it will be effectively replaced by `finalTimeout`.